docet-core/src/main/java/docet/engine/DocetManager.java

```diff
@@ -1239,12 +1239,11 @@ private DocetDocumentParser getParserForFormat(final DocetDocFormat format) thro
     private static final boolean isPDFGenerationLibraryPresent() {
         ClassLoader loader = DocetManager.class.getClassLoader();
         try {
-            // Check a class from com.itextpdf:itextpdf
-            loader.loadClass("com.itextpdf.text.pdf.PdfWriter");
-
-            // Check a class from com.itextpdf.tool:xmlworker
-            loader.loadClass("com.itextpdf.tool.xml.pipeline.html.ImageProvider");
+            // Check a class from com.lowagie:itext
+            loader.loadClass("com.lowagie.text.pdf.PdfWriter");
 
+            // Check a class from org.xhtmlrenderer:flying-saucer-pdf
+            loader.loadClass("org.xhtmlrenderer.pdf.ITextOutputDevice");
             LOGGER.log(Level.SEVERE, "PDF generation enabled");
```

Comment:
change to info or finest ? this is no more a real problem

Reply:
Docet can "work" but cannot produce any pdf in such state, signalling it to log one time at startup 
 can be useful, probabily is better to move down to warning or info, finest is usually filtered out from logs :(

Reply:
Uh... well change just the "ok you can generate pdf" is correct
:+1: